loader_wsock32_proxy/loader_wsock32_proxy.vcxproj

```
@@ -60,15 +60,15 @@
       <ConformanceMode>true</ConformanceMode>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
-      <LanguageStandard>stdcpp17</LanguageStandard>
+      <LanguageStandard>stdcpp20</LanguageStandard>
       <AdditionalIncludeDirectories>..\NorthstarDedicatedTest\</AdditionalIncludeDirectories>
```

```suggestion
      <AdditionalIncludeDirectories>..\NorthstarDLL\</AdditionalIncludeDirectories>
```

This should probably be updated as well :P

---

neither should be necessary, didn't notice we still had the include directories in the project file tho